packages/core/capabilities/core-capabilities-server-internal/src/resolve_capabilities.ts

```diff
@@ -27,12 +28,14 @@ export const getCapabilitiesResolver =
     applications: string[],
     useDefaultCapabilities: boolean
   ): Promise<Capabilities> => {
-    return resolveCapabilities(
-      capabilities(),
-      switchers(),
-      request,
-      applications,
-      useDefaultCapabilities
+    return withSpan({ name: 'resolve capabilities', type: 'capabilities' }, () =>
```

Comment:
I assumed the utilities are tested at the package level and that we don't need to test that they're functioning correctly here.